src/seedu/addressbook/Main.java

```diff
@@ -55,7 +55,7 @@ private void start(String[] launchArgs) {
             this.addressBook = storage.load();
             ui.showWelcomeMessage(VERSION, storage.getPath());
 
-        } catch (InvalidStorageFilePathException | StorageOperationException e) {
+        } catch (InvalidStorageFilePathException | StorageOperationException | FileNotFoundException e) {
```

Comment:
this need not be only on `init`. the file can go missing anytime during the program execution.
hence, i would suggest a better way of handling the file not found exception here.